jellyfin.tf

```diff
@@ -252,7 +252,7 @@ resource "aws_instance" "jellyfin_server" {
 
   provisioner "remote-exec" {
     inline = [
-      "EBS_DEVICE_NAME=$(lsblk | grep ${var.EBS_MEDIA_VOLUME_SIZE}G | awk '{print $1}')",
+      "EBS_DEVICE_NAME=$(lsblk -b | grep $((1073741824 * ${var.EBS_MEDIA_VOLUME_SIZE})) | awk '{print $1}')",
```

Comment:
* `-b` tells `lsblk` to print sizes in number of bytes
* There are 1073741824 bytes in a GiB, so this multiplication operation effectively allows us to use the existing units for `EBS_MEDIA_VOLUME_SIZE`.